examples/density_baseline_demo.ipynb

```diff
@@ -0,0 +1,252 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Demo of simple density-based imitation learning baselines\n",
+    "\n",
+    "This demo shows how to train a `Pendulum` agent (exciting!) with our simple density-based imitation learning baselines. `DensityTrainer` has a few interesting parameters, but the key ones are:\n",
+    "\n",
+    "1. `density_type`: this governs whether density is measured on $(s,s')$ pairs (`db.STATE_STATE_DENSITY`), $(s,a)$ pairs (`db.STATE_ACTION_DENSITY`), or single states (`db.STATE_DENSITY`).\n",
+    "2. `is_stationary`: determines whether a separate density model is used for each time step $t$ (`False`), or the same model is used for transitions at all times (`True`).\n",
+    "3. `standardise_inputs`: if `True`, each dimension of the agent state vectors will be normalised to have zero mean and unit variance over the training dataset. This can be useful when not all elements of the demonstration vector are on the same scale, or when some elements have too wide a variation to be captured by the fixed kernel width (1 for Gaussian kernel).\n",
+    "4. `kernel`: changes the kernel used for non-parametric density estimation. `gaussian` and `exponential` are the best bets; see the [sklearn docs](https://scikit-learn.org/stable/modules/density.html#kernel-density) for the rest."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "#%load_ext autoreload\n",
+    "#%autoreload 2\n",
+    "\n",
+    "import pprint\n",
+    "\n",
+    "from imitation.algorithms import density_baselines as db\n",
+    "from imitation.data import types\n",
+    "from imitation.util import util\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "outputs": [],
+   "source": [
+    "# Set FAST = False for longer training. Use True for testing and CI.\n",
+    "FAST = True\n",
+    "\n",
+    "if FAST:\n",
+    "    N_VEC = 1\n",
+    "    N_TRAJECTORIES = 1\n",
+    "    N_ITERATIONS = 1\n",
+    "    N_RL_TRAIN_STEPS = 100\n",
+    "else:\n",
+    "    N_VEC = 8\n",
+    "    N_TRAJECTORIES = 10\n",
+    "    N_ITERATIONS = 100\n",
+    "    N_RL_TRAIN_STEPS = int(1e5)"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n",
+     "is_executing": false
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "pycharm": {
+     "is_executing": false
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "text": [
+      "Using cuda device\n"
+     ],
+     "output_type": "stream"
+    }
+   ],
+   "source": [
+    "env_name = 'Pendulum-v0'\n",
+    "env = util.make_vec_env(env_name, N_VEC)\n",
+    "rollouts = types.load(\"../tests/data/expert_models/pendulum_0/rollouts/final.pkl\")\n",
```

Comment:
See comments for `tests/test_examples.py` re: relative paths. Bugs me less for notebooks though which are meant to be hacky.

Reply:
At least 0.5 years ago, Jupyter notebook always runs the notebook with the notebook's parent directory as the working directory (I tried in vain to configure this to be the project root instead). So this should be more likely to be fine than the `tests/test_examples.py` case.